package-lock.json

```diff
@@ -970,9 +970,9 @@
       }
     },
     "node_modules/@apollo/client": {
-      "version": "3.4.9",
-      "resolved": "https://registry.npmjs.org/@apollo/client/-/client-3.4.9.tgz",
-      "integrity": "sha512-1AlYjRJ/ktDApEUEP2DqHI38tqSyhSlsF/Q3fFb/aCbLHQfcSZ1dCv7ZlC9UXRyDwQYc0w23gYJ7wZde6W8P4A==",
+      "version": "3.4.11",
```

Comment:
can revert this file, no package json changes mean no package lock changes should be pulled in.

Reply:
how do I revert it? I basically had checked in the old version in a commit. So, had to regenerate the package.lock

Reply:
git checkout unless I'm misunderstanding? `git checkout origin/main package-lock.json`

Reply:
my git knowledge is not good. I realized it when I messed up my angular repo PR (twice). Such an embarrassment.

Reply:
done